packages/duf/src/evo/data_converters/duf/common/duf_wrapper.py

```diff
@@ -0,0 +1,242 @@
+# ruff: noqa: E402
+
+import os
+import platform
+from collections import defaultdict
+
+if not platform.system() == "Windows":
+    raise RuntimeError("This script is only supported on Windows.")
+
+if not os.path.exists(r"C:\Program Files\Deswik"):
+    raise RuntimeError("Deswik.Suite is not installed. Please install Deswik.Suite to run this script.")
+
+installs = [pth for pth in os.listdir(r"C:\Program Files\Deswik") if "Suite" in pth]
+if not installs:
+    raise RuntimeError("Deswik.Suite is not installed. Please install Deswik.Suite to run this script.")
+
+
+# Sort by version
+def by_version(path):
+    version = path.split(" ")[-1]
+    year, month = version.split(".")
+    return int(year), int(month)
+
+
+import evo.logging
+
+logger = evo.logging.getLogger("data_converters")
+
+newest = os.path.join(r"C:\Program Files\Deswik", sorted(installs, key=by_version)[-1])
+logger.debug("Looking for Deswik DLLs in: %s", newest)
+
+import sys
+
+sys.path.append(newest)
+
+import clr
+
+clr.AddReference("Deswik.Duf")
+clr.AddReference("Deswik.Entities")
+clr.AddReference("Deswik.Entities.Cad")
+clr.AddReference("Deswik.Serialization")
+
+# Some imports below are unused but imported here to ensure the setup code above has run
+from Deswik.Duf import CompressionMethod, DufImplementation, FilterCriteria, NotDufFileException
+from Deswik.Entities import BaseEntity
+from Deswik.Entities.Cad import Activator, Category, Polyface, Polyline, Upgrader  # noqa: F401
+from Deswik.Serialization import GuidReferences
+from System import Guid, NullReferenceException
+from System.Collections.Generic import List
+
+
+class InvalidDufFileException(ValueError):

```

Comment:
```suggestion
class InvalidDUFFileException(ValueError):
```

If you could rename all `Duf` to `DUF` that would be great. There's a PEP standard somewhere that says that acronyms should be uppercase.